app/middleware/regen.sh

```diff
@@ -0,0 +1 @@
+protoc --js_out=import_style=commonjs,binary:./walletrpc --grpc_out=./walletrpc --plugin=protoc-gen-grpc=$(which grpc_node_plugin) ./api.proto
```

Comment:
Could you put a comment to say what this script is for.  It is 100% obvious now, but a few months from now....